igor-web/src/main/java/com/netflix/spinnaker/igor/config/DeliveryConfigProperties.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.igor.config;
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@Data
+@ConfigurationProperties
+public class DeliveryConfigProperties {
+  String manifestBasePath = ".netflix";
```

Comment:
This is configurable now, but defaults to `.netflix`.

Reply:
nit: can we have it default to something that isn't `.netflix`? IMO our defaults should be appropriate for any company, and if there's no default that's appropriate for any company it should be a required bit of config vs. optional.

Reply:
Absolutely! I was thinking one thing and wrote another. Twice! 🤦‍♂️ It was supposed to be `.spinnaker`. Will make the change!